mtp_api/apps/security/serializers.py

```diff
@@ -220,9 +224,92 @@ def update(self, instance, validated_data):
         return super().update(instance, validated_data)
 
 
+class CheckAutoAcceptRuleStateSerializer(serializers.ModelSerializer):
+    active = serializers.BooleanField(required=False)
+    added_by = BasicUserSerializer(read_only=True)
+
+    class Meta:
+        model = CheckAutoAcceptRuleState
+        fields = [
+            'added_by',
+            'active',
+            'reason',
+            'created',
+            'auto_accept_rule'
+        ]
+        read_only_fields = (
+            'id',
+            'auto_accept_rule'
+        )
+
+    def validate(self, attrs):
+        if self.instance and attrs.get('active') is None:
+            raise ValidationError({
+                'active': ValidationError(_('On update, states[0].active must be specified'))
```

Comment:
related to the above point about why `active` isn't always required. also with this version, you can post True two successive times.